cmake/test/gtest.cmake

```diff
@@ -300,6 +303,12 @@ function(catkin_find_google_test_source gtest_path googletest_path
   endif()
 endfunction()
 
+get_property(ENABLED_LANGUAGES_LIST GLOBAL PROPERTY ENABLED_LANGUAGES)
+list(FIND ENABLED_LANGUAGES_LIST "CXX" _cxx_index)
+if ("${_cxx_index}" EQUAL -1) # if C++ isn't an enabled language
+  return()
+endif()
+
```

Comment:
This logic can never be triggered. The macro `_generate_function_if_no_cxx_language` already contains a `return()` statement which ends the file when it is invoked in line 4.

Reply:
Think I forgot to remove this from the first attempt, before it was using `_generate_function...`. Thanks